metadata.json

```diff
@@ -9,12 +9,12 @@
   "issues_url": "https://github.com/crayfishx/puppet-rsyslog/issues",
   "dependencies": [
     {"name":"puppetlabs-stdlib","version_requirement":">= 1.0.0 < 5.0.0"},
-    {"name":"puppetlabs-concat","version_requirement":">= 2.0.0 < 3.0.0"}
+    {"name":"puppetlabs-concat","version_requirement":">= 2.0.0 < 5.0.0"}
   ],
   "requirements": [
     { 
       "name": "puppet",
-      "version_requirement": ">= 4.0.0 < 5.0.0"
+      "version_requirement": ">= 4.4.0 < 6.0.0"
```

Comment:
It would be nice to keep support for the whole 4.x range - I can't see what feature is used that is unsupported < 4.4.... ?